src/Board.hpp

```diff
@@ -1,9 +1,12 @@
 #pragma once
 #include "json.hpp"
+#include "FieldsEnum.hpp"
 #include <vector>
 
 using Line = std::vector<int>;
 using Lines = std::vector<Line>;
+using BLine = std::vector<FieldsEnum>;
+using BLines = std::vector<BLine>;
```

Comment:
BLine i BLines ment to be BoardLines, new type for board_ stored in Board class. This declaration is temporary only for test purpose. It will be properly declared in US2.2.